tls/s2n_record_read.c

```diff
@@ -206,13 +226,25 @@ int s2n_record_parse(struct s2n_connection *conn)
 
         GUARD(cipher_suite->cipher->io.aead.decrypt(session_key, &iv, &aad, &en, &en));
         break;
+    case S2N_COMPOSITE:
+        ne_check(en.size, 0);
+        eq_check(en.size % iv.size,  0);
```

Comment:
This needn't be true for all potential composite cipher types. O.k. for now though  I think ... works for AES-CBC